pkg/translator/azurelogs/testdata/frontdooraccesslog/valid_1.json

```diff
@@ -0,0 +1,47 @@
+{
+    "records":[
+        {
+            "time":"2025-04-24T13:14:28.0000000Z",
+            "resourceId":"/SUBSCRIPTIONS/OPENTELEMETRY-AZURE-SUB/RESOURCEGROUPS/OPENTELEMETRY-FRONTDOOR/PROVIDERS/MICROSOFT.CDN/PROFILES/OPENTELEMETRY-FRONTDOOR-PROFILE",
+            "category":"FrontDoorAccessLog",
+            "operationName":"Microsoft.Cdn/Profiles/AccessLog/Write",
+            "properties":{
+                "trackingReference":"20250424T131428Z-17587c8c466d76czhC1PARprs40000000q8g00000000d67w",
+                "httpMethod":"GET",
+                "httpVersion":"2.0.0.0",
+                "requestUri":"https://opentelemetry-test-fmagg0exgdcfhefq.z01.azurefd.net:443/",
+                "sni":"opentelemetry-test-fmagg0exgdcfhefq.z01.azurefd.net",
+                "requestBytes":"60",
+                "responseBytes":"1624",
+                "userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:137.0) Gecko/20100101 Firefox/137.0",
+                "clientIp":"2001:1c00:3280:6700:fbfa:bf04:1296:ebfc",
+                "clientPort":"55262",
+                "socketIp":"2001:1c00:3280:6700:fbfa:bf04:1296:ebfc",
+                "timeToFirstByte":"0.035",
+                "timeTaken":"0.035",
+                "requestProtocol":"HTTPS",
+                "securityProtocol":"TLS 1.3",
+                "rulesEngineMatchNames":[ ],
+                "httpStatusCode":"200",
+                "httpStatusDetails":"200",
+                "pop":"PAR",
+                "cacheStatus":"CONFIG_NOCACHE",
+                "errorInfo":"NoError",
+                "ErrorInfo":"NoError",
+                "result":"N/A",
+                "endpoint":"opentelemetry-test-fmagg0exgdcfhefq.z01.azurefd.net",
+                "routingRuleName":"opentelemetry-frontdoor-route",
+                "hostName":"opentelemetry-test-fmagg0exgdcfhefq.z01.azurefd.net",
+                "originUrl":"https://opentelemetry-app.azurewebsites.net:443/",
+                "originIp":"23.100.1.29:443",
+                "originName":"opentelemetry-app.azurewebsites.net:443",
+                "originCryptProtocol":"N/A",
+                "referer":"",
+                "clientCountry":"Netherlands",
```

Comment:
We're no longer translating this to anything. Is that expected?

I think we should ideally set `client.geo.country.iso_code`, which will require a lookup table. Should we open a followup issue for this?

Reply:
I remembered this from the AWS WAF logs :) Yes, my issue was with the code. I will try to see if there is an "easy" way to get the iso code